test/e2e/framework/framework.go

```diff
@@ -655,7 +655,7 @@ func (f *Framework) ForwardPort(t *testing.T, ns, svc string, port int) (string,
 func Poll(interval, timeout time.Duration, f func() error) error {
 	var lastErr error
 
-	err := wait.PollUntilContextTimeout(context.Background(), interval, timeout, false, func(context.Context) (bool, error) {
```

Comment:
All of the runs I checked were timing out, I think this is the change that may make a difference. Do you think we need to have the rest as well? (as I'm not seeing any flaky tests)